pkg/build/krel.go

```diff
@@ -48,6 +48,7 @@ func (rpb *ReleasePushBuild) Stage(version string) error {
 	}
 
 	// currently krel requires to be run from kubernetes root
+	// should support passing working directory
```

Comment:
we should probably help patch this as well, having to `os.Chdir` is brittle, particularly for "library" code.

Reply:
created https://github.com/kubernetes/release/issues/1891